Сетка страниц и крупных блоков на флексах

Fallen911 opened the pull request:
---
:mortar_board: [Сетка страниц и крупных блоков на флексах](https://up.htmlacademy.ru/htmlcss/34/user/2096277/tasks/7)

:boom: https://htmlacademy-htmlcss.github.io/2096277-sedona-34/5/

Fallen911 committed:
добавляет часть гридов и флексов в индекс

Fallen911 committed:
делает сетки страниц на флексах

senchkim commented on catalog.html:
```diff
@@ -6,6 +6,7 @@
         <title>Каталог</title>
```
https://validator.w3.org/nu/#textarea

Ошибки типа `error` в валидаторе, их нужно исправить

senchkim commented on styles/styles.css:
```diff
@@ -42,14 +52,86 @@ body {
 
+.main-container {
+    display: flex;
+
```
лишний отступ

senchkim commented on styles/styles.css:
```diff
@@ -42,14 +52,86 @@ body {
+.container-header {
+    display: inline;
+}
+.navigation {
```
отступ между свойствами

senchkim commented on styles/styles.css:
```diff
@@ -42,14 +52,86 @@ body {
+}
+
+.header-navigation {
+
```
пустой класс так и задумано?

senchkim commented on styles/styles.css:
```diff
@@ -42,14 +52,86 @@ body {
 .advanteges-list {
+    display: flex;
+    flex-wrap: wrap;
+    
```
пустая строка?

senchkim commented on styles/styles.css:
```diff
@@ -42,14 +52,86 @@ body {
 }
 
+.description-1 {
+font-weight: 700;
```
Форматирование поехало

senchkim commented on styles/styles.css:
```diff
@@ -42,14 +52,86 @@ body {
+}
+
+.description-2 {
+font-size: 22px;
```
Форматирование поехало

senchkim commented on styles/styles.css:
```diff
@@ -178,12 +269,20 @@ a[href^="tel:"] {
+    margin-left: auto;
 }
 
 .reset {
```
на 289 строке, нет запятой

senchkim commented on styles/styles.css:
```diff
@@ -227,4 +326,79 @@ a[href^="tel:"] {
+
+.resoult {
+    width: 700px;
+  
```
лишняя пустая строка

senchkim commented on styles/styles.css:
```diff
@@ -227,4 +326,79 @@ a[href^="tel:"] {
     text-decoration: none;
+}
+
+.container {
```
по идее это должно идти в стилях после стилей `body`

Fallen911 committed:
Исправляет ошибки

Извиняюсь, что в первый раз сырой проект  залил. Неделя выпала из жизни, приходиться наверстывать

Fallen911 commented:
Исправил ошибки
